Update version-control-basics.md (#5808)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/git/version-control-basics.md

| Open pull request | This allows you to open a pull request in Git for peers to review changes before merging into the base branch. |
| Pull changes from main | This option is available if you are on any local branch that is behind the remote version of the base branch or the remote version of the branch that you're currently on. |
| Pull from remote | This option is available if you’re on the local base branch and changes have recently been pushed to the remote version of the branch. Pulling in changes from the remote repo allows you to pull in the most recent version of the base branch. |

> | Rollback to remote | Reset changes your repository directly from the Cloud IDE. You can reset your repository back to a fresh clone from your remote. To do this, click on the three dot ellipsis in the bottom right-hand side of the IDE and select **Rollback to remote**.  |

Reviewer: Suggestion to give customers more pause before making the decision based on the convo here: https://dbt-labs.slack.com/archives/C012TTNN0JK/p1719603672914079?thread_ts=1719603609.586689&cid=C012TTNN0JK
```suggestion
| Rollback to remote | Reset changes to your repository directly from the Cloud IDE. You can rollback your repository back to an earlier clone from your remote. To do this, click on the three dot ellipsis in the bottom right-hand side of the IDE and select **Rollback to remote**.  |
```

> | Reclone Your Repository | Reclone your repository directly from the Cloud IDE. You can reset your repository back to a fresh clone from your remote. To do this, click on the three horizontal dots on the bottom right-hand side (next to the green **Ready** text), then click **Rollback to remote**.  |

Reviewer 2: ```suggestion
| Rollback to remote | Reset changes your repository directly from the Cloud IDE. You can reset your repository back to a fresh clone from your remote. To do this, click on the three dot ellipsis in the bottom right-hand side of the IDE and select **Rollback to remote**.  |
```
